### ISSUE TITLE
omni_send result : "Error with selected inputs for the send transaction" 

### ISSUE THREAD

**Issue by ssssssu12** [OmniLayer/omnicore#823]
Hello,
This is my address : 2N6mfw33o6vLtDKDEXg92jQsseyveevnpsX
balance : 
<pre><code>curl -H 'Content-Type: application/json' --user 'aaa:aaa' -X POST --data-binary '{"jsonrpc": "2.0", "id":"omni_test", "method": "omni_getbalance", "params": ["2N6mfw33o6vLtDKDEXg92jQsseyveevnpsX", 2] }' http://127.0.0.1:38832
{"result":{"balance":"10.00000000","reserved":"0.00000000","frozen":"0.00000000"},"error":null,"id":"omni_test"}</code></pre>
<img width="1174" alt="2018-11-06 10 32 21" src="https://user-images.githubusercontent.com/10285287/48037368-49e3ea80-e1af-11e8-8877-d19af3c54832.png">

and, call omni_send : 
<pre><code>curl -H 'Content-Type: application/json' --user 'aaa:aaa' -X POST --data-binary '{"jsonrpc": "2.0", "id":"omni_test", "method": "omni_send", "params": ["2N6mfw33o6vLtDKDEXg92jQsseyveevnpsX", "mgvooSaswzDNNZP6rQsEsjmpGbDLkRzTcQ", 2, "0.1"] }' http://127.0.0.1:38832
{"result":null,"error":{"code":-206,"message":"Error with selected inputs for the send transaction"},"id":"omni_test"}</code></pre>

why omni_send error "Error with selected inputs for the send transaction" ?

**Comment by santos177** [comment]
Hi @ssssssu12:
The only trusted information about the bitcoins availables that you have for your address is watching the unspent transaction outputs (using omnicore synchronized). If you don't have enough money there, you are going to have the -206 error . Please use this: http://chainquery.com/bitcoin-api/listunspent

**Comment by santos177** [comment]
Hi @ssssssu12:   Is omnicored synchronized?  Check the block heigh using omni_getinfo.

**Comment by ssssssu12** [comment]
Yes.. block height same https://live.blockcypher.com/btc-testnet/
<pre><code>curl -H 'Content-Type: application/json' --user 'aaa:aaa' -X POST --data-binary '{"jsonrpc": "2.0", "id":"omni_test", "method": "omni_getinfo", "params": [] }' http://127.0.0.1:38832
{"result":{"omnicoreversion_int":30001000,"omnicoreversion":"0.3.1","mastercoreversion":"0.3.1","bitcoincoreversion":"0.13.2","block":1442210,"blocktime":1541473589,"blocktransactions":1,"totaltrades":15601,"totaltransactions":67570,"alerts":[]},"error":null,"id":"omni_test"}</code></pre>
but different balance. what's the problem..?

**Comment by ssssssu12** [comment]
@santos177 Thanks!
https://live.blockcypher.com/btc-testnet/address/2N6mfw33o6vLtDKDEXg92jQsseyveevnpsX/ 
<pre><code>./omnicore-cli "omni_getinfo"
{
  "omnicoreversion_int": 30001000,
  "omnicoreversion": "0.3.1",
  "mastercoreversion": "0.3.1",
  "bitcoincoreversion": "0.13.2",
  "block": 1442227,
  "blocktime": 1541478831,
  "blocktransactions": 0,
  "totaltrades": 15601,
  "totaltransactions": 67578,
  "alerts": [
  ]
}</pre></code>
<pre><code>./omnicore-cli "listunspent" 0 999999 '["2N6mfw33o6vLtDKDEXg92jQsseyveevnpsX"]'
[
]</pre></code>

still diff.. what's the problem ?

**Comment by santos177** [comment]
Maybe is because you are using a multisig address , watch this:  #810

**Comment by ssssssu12** [comment]
Um.. okay! 
I'll check that. Thanks!

**Comment by livingrock7** [comment]
omni_send and couple other write methods are "not found" on my omni node. Can anyone please advise?
